bindings/wasm/bindings.cpp

```diff
@@ -178,6 +178,7 @@ EMSCRIPTEN_BINDINGS(whatever) {
       .function("_Scale", &Manifold::Scale)
       .function("_Decompose", &Manifold::Decompose)
       .function("isEmpty", &Manifold::IsEmpty)
+      .function("isManifold", &Manifold::IsEmpty)
```

Comment:
Let's skip this one; we discussed elsewhere removing it from the C++ API too; it's a bit of a misnomer at this point anyway, and it was only ever for internal testing.

Reply:
Reverted. Also, I didn't notice this at the time, but I forgot to change the function reference 😬